friendsofsymfony/user-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "FOS\\UserBundle\\FOSUserBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "MAILER_USER": ""
```

Comment:
I think `MAILER_USER` is inconsistent with possible configuration value. I think `MAILER_SENDER` would probably be more appropriate. A "user" option is more to configure the mailer with a user.

Reply:
I agree